src/CodeGen_C.cpp

```diff
@@ -2053,7 +2053,7 @@ void CodeGen_C::visit(const Call *op) {
         if (op->args.empty()) {
             internal_assert(op->type.handle_type);
             // Add explicit cast so that different structs can't cache to the same value
-            rhs << "(" << print_type(op->type) << ")(NULL)";
+            rhs << "(" << print_type(op->type) << ")(nullptr)";
```

Comment:
Do we want this one?

Reply:
Our generated C++ code already requires C++11 (or later), so, sure

Reply:
Isn't this used by e.g. OpenCL and it's variety of compilers of varying quality?

I'd suggest simply using 0 here.

Reply:
good point. (though I wouldn't be surprised at a barf at that from dubious OpenCL compilers too.) Maybe swapping back to NULL would be safest since we know it works.

Reply:
Swapped back to NULL.